File: 1009.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

var nome = lines.shift()
var salarioFixo = parseFloat(lines.shift());
var totalVendas = parseFloat(lines.shift());
var salarioTotal = salarioFixo + (totalVendas * 0.15)

console.log("TOTAL = R$ " + salarioTotal.toFixed(2))

File: Java/1009.java
import java.util.Scanner;

public class 1009 {

  public static void main(String[] args)  {
    Scanner scanner = new Scanner(System.in);

    String nome = scanner.next();
    double salarioFixo = scanner.nextDouble();
    double totalVendas = scanner.nextDouble();

    double salarioTotal = salarioFixo + (totalVendas * 0.15);
    
    String salarioTotalFormatado = String.format("%.2f", salarioTotal);

    System.out.println("TOTAL = R$ " + salarioTotalFormatado);
  }
}